plugin/src/main/java/io/github/guegse/foreachstream/plugin/ASTHelpers.java

```diff
@@ -0,0 +1,124 @@
+package io.github.guegse.foreachstream.plugin;
+
+import com.sun.source.tree.*;
+import com.sun.tools.javac.code.Symbol;
+import com.sun.tools.javac.code.Type;
+import com.sun.tools.javac.tree.JCTree;
+import com.sun.tools.javac.tree.JCTree.*;
+
+public class ASTHelpers {
```

Comment:
Please add license information (Apache license) and a comment that this was taken from errorprone